low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -365,8 +365,10 @@ def _generate_host_update(self):
             if not self.host_fault:
                 self.host_fault = True
                 # Create the disk space data message and hand it over to the egress processor to transmit
-                logger.warning("Host Memory usage increased to {}%, beyond configured threshold of {}%".\
-                    format(self._node_sensor.total_memory["percent"], self._host_memory_usage_threshold))
+                fault_event = "Host memory usage increased to %s, beyond configured threshold of %s" \

```

Comment:
OK i get it, these will be in event. Lets put these in LDR R2 specification 'Messages' tab & get it reviewed. Mark it that these are being introduced with tkt number